tests/sentry/integrations/tasks/test_sync_assignee_outbound.py

```diff
@@ -35,19 +37,23 @@ def setUp(self):
             },
         )
 
-    @mock.patch("sentry.integrations.utils.metrics.EventLifecycle.record_event")
-    @mock.patch.object(ExampleIntegration, "sync_assignee_outbound")
-    def test_syncs_outbound_assignee(self, mock_sync_assignee, mock_record_event):
-        external_issue = self.create_integration_external_issue(
+        # Create a shared external issue for most tests
+        self.external_issue = self.create_integration_external_issue(
```

Comment:
This was repeated in nearly every single test, so I hoisted it into the setup instead.